# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvi/types.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

One more type conflict ...

  - Rene Rebe <rene@exactcode.de>

diff -ur nvi.1.49/PORT/linux/db/hash/hash.c nvi.1.49-fixed/PORT/linux/db/hash/hash.c
--- nvi.1.49/PORT/linux/db/hash/search.h	1993-06-05 00:22:44.000000000 +0200
+++ nvi.1.49-fixed/PORT/linux/db/hash/search.h	2005-02-02 13:52:06.221408376 +0100
@@ -46,6 +46,6 @@
 	FIND, ENTER
 } ACTION;
 
-int	 hcreate __P((unsigned int));
+int	 hcreate __P((u_int));
 void	 hdestroy __P((void));
 ENTRY	*hsearch __P((ENTRY, ACTION));
--- nvi.1.49/ex/filter.c.vanilla	2005-02-02 15:21:01.656298984 +0100
+++ nvi.1.49/ex/filter.c	2005-02-02 15:21:10.680927032 +0100
@@ -342,7 +342,6 @@
 	const char *cmd;
 	int okpipe;
 {
-	extern const char *const sys_siglist[];
 	size_t len;
 	int nf, pstat;
 	char *p;
